editor/actions.js

```diff
@@ -134,6 +134,12 @@ export function clearSelectedBlock() {
 	};
 }
 
+export function toggleSelectionDisable() {
+	return {
+		type: 'TOGGLE_SELECTION_DISABLE',
+	};
+}
```

Comment:
I'd go for just _toggle selection_ (in both the action name and action type). Maybe even _toggle block selection_.